index.js

```diff
@@ -92,9 +162,14 @@ const promo20 = users => {
 
 const addActive = users => {
   // Your code goes here ...
+for (let i in users){
+  users[i].isActive = true
+}
+return users
```

Comment:
This function immediately is changing the values you are passing onto it.

In functions, unless said otherwise, try to create ahabit that you return a totally new value.
For example
```js
const addActive = users => {
return users.map(user => {
return {
...user, //here you are getting the whole user object copied
isActive: true
}
})
}
```
After this function you get an array of every user active but you get a totally new value.

Reply:
So, I'm changing the values of the original array and by using map() it returns a new array?